contributing/config/README.md

```diff
@@ -1,3 +1,53 @@
 # Agent Configuration
 
-- [Checklist for adding a new field](./checklist-adding-config-fields.md)
+The [Agent Configuration] is the primary mechanism for configuring Consul. Agent
+Configuration also allows for specifying Config Entries, Services, and Checks that
```

Comment:
would it be helpful to link out to the main docs on these topics for good measure (similar to what is there for Agent Configuration).